src/components/Breadcrumb/index.tsx

```diff
@@ -15,26 +16,37 @@ export const BreadCrumb: React.FC<Props> = ({
   isTask = false
 }) => {
   return (
-    <Breadcrumb>
-      <Breadcrumb.Item>
+    <Breadcrumb style={{ marginBottom: "25px" }}>
+      <StyledBreadcrumItem link={true}>
```

Comment:
I get this warning in my console.
`Warning: Received `true` for a non-boolean attribute `link`.`